docs/V1/Customer/get_respondents.md

```diff
@@ -162,7 +165,7 @@ first_name       | Prénom du répondant
 last_name        | Nom du répondant 
 candidate_link   | Lien vers la fiche du répondant (utilisateur connecté sur Scoringline) 
 synthesis_link   | Lien vers la synthèse PDF du répondant (utilisateur connecté sur Scoringline) 
-status           | Statut (état) du répondant
+status           | Statut (état) du répondant (int)
```

Comment:
@tontonjulien I guess it would be nice to have another column with types specified in there :slightly_smiling_face: